src/constants/labels.json

```diff
@@ -161,9 +161,15 @@
     "words": "Wörter",
     "camera": {
       "noAuthorization": {
-        "description": "Kamerazugriff erforderlich. Öffnen Sie die Einstellungen, tippen Sie auf Berechtigungen und tippen Sie auf Erlauben.",
-        "settings": "Einstellungen"
+        "description": "Kamerazugriff erforderlich. Öffnen Sie die Einstellungen, tippen Sie auf Berechtigungen und tippen Sie auf Erlauben."
       }
+    },
+    "audio": {
+      "noAuthorization": {
+        "description": "Mikrofonzugriff erforderlich. Öffnen Sie die Einstellungen, tippen Sie auf Berechtigungen und tippen Sie auf Erlauben."
```

Comment:
I see this was wrong also for images, but lunes uses "du" not "Sie". Please adjust.